controllers/metal3machine_controller.go

```diff
@@ -68,6 +68,7 @@ type Metal3MachineReconciler struct {
 // +kubebuilder:rbac:groups=cluster.x-k8s.io,resources=kubeadmcontrolplanes,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups="",resources=events,verbs=get;list;watch;create;update;patch
 // +kubebuilder:rbac:groups="",resources=secrets,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups="",resources=configmaps,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Do we need all verbs ? if so, replaced by *
Also, should this scoped to some namespace ?
```suggestion
// +kubebuilder:rbac:groups="",resources=configmaps,verbs=*
```

Reply:
We did not specify  namespaces for other resources. If you think this is the right way, I can add

Reply:
@Xenwar are we sure we can replace verbs with *, if so can you please reference to the example where it has been used? I have not seen it in practice so far and [kubeuilder](https://book.kubebuilder.io/reference/markers/rbac.html) book does not seem to mention that

Reply:
It is a valid syntax. You can use `* ` for resources or verbs. Here is one example for resources.
https://github.com/kubernetes-sigs/cluster-addons/blob/b8b26d8aa73c917c00ed58871de87bd8a3b6c56a/dashboard/controllers/dashboard_controller.go#L91

But, I agree with making it like other controllers. So, we can keep the list of verbs.